examples/bnb-example/src/app/bnb.tx.example.ts

```diff
@@ -0,0 +1,28 @@
+import { TatumBnbSDK } from '@tatumio/bnb'
+import { Currency } from '@tatumio/api-client'
+
+export async function bnbTxExample() {
+  const bnbSDK = TatumBnbSDK({ apiKey: 'PLACE-YOUR-API-KEY-HERE' })
```

Comment:
we've discussed to place this key everywhere '75ea3138-d0a1-47df-932e-acb3ee807dab' in examples